x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -114,6 +114,27 @@ export async function closePointInTime(esClient: ElasticsearchClient, pitId: str
   }
 }
 
+export async function getAgentTags(esClient: ElasticsearchClient): Promise<string[]> {
+  try {
+    const result = await esClient.search<{}, { tags: { buckets: Array<{ key: string }> } }>({
+      index: AGENTS_INDEX,
+      size: 0,
+      aggs: {
+        tags: {
+          terms: { field: 'tags' },
```

Comment:
We will need to use the [size](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-size) parameter here, is there a max number of unique tags we will allow?

Reply:
good catch! I will set it to a higher number, I think generally we won't have too many unique tags.

Reply:
added size parameter, so now more than 10 tags are coming back as expected:
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/90178898/180222428-2645d367-39cb-44a1-8b32-8ec499f1cb50.png">